test/models/registration_test.rb

```diff
@@ -1,7 +1,64 @@
+# frozen_string_literal: true
+
 require 'test_helper'
 
 class RegistrationTest < ActiveSupport::TestCase
   test 'factory creates valid registration' do
     assert build(:registration).valid?
   end
+
+  test 'accommodated_duration_minutes with no accommodation' do
+    reg = build(:registration)
+    exam = reg.exam
+    assert_equal exam.duration, reg.accommodated_duration
+  end
+
+  test 'accommodated_start_time with no accommodation' do
+    reg = build(:registration)
+    exam = reg.exam
+    assert_equal reg.accommodated_start_time, exam.start_time
+  end
+
+  test 'accommodated_extra_duration with no accommodation' do
+    reg = build(:registration)
+    assert_equal reg.accommodated_extra_duration, 0
+  end
+
+  test 'accommodated_end_time with no accommodation' do
+    reg = build(:registration)
+    exam = reg.exam
+    assert_equal reg.accommodated_end_time.to_i, exam.end_time.to_i
```

Comment:
https://chriskottom.com/blog/2018/01/comparing-timestamps-in-rails/

Reply:
Ok...  It seems like, in these test cases at least, all the timestamps are `TimeWithZone` objects, and since we're computing the various times off one another via arithmetic, the straightforward comparisons at the `TimeWithZone` class should work just fine.  (They pass all the tests for me when I try dropping the `to_i` calls.)

Reply:
Interesting... they are working for me as well but definitely were not..

Reply:
Yeah.  I suspect it was because of inconsistent usage of .seconds, .minutes, and other math.  I'm rather happier for them to be working at the `TimeWithZone` type, and if they regress, that'll be an interesting inconsistency.